browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -1579,29 +1618,52 @@ void RewardsDOMHandler::GetExternalWallet(const base::ListValue* args) {
       &RewardsDOMHandler::OnGetExternalWallet, weak_factory_.GetWeakPtr()));
 }
 
-void RewardsDOMHandler::OnGetExternalWallet(
+base::Value RewardsDOMHandler::ExternalWalletToValue(
```

Comment:
this method should either be split up or renamed because it does more then just convert the ExternalWallet to base::Value